draft-ietf-shmoo-online-meeting.md

```diff
@@ -120,143 +175,150 @@ one of those regions.
-even though time zones were rotated. This indicates that online meetings support a more easy and therefore potentially
-broader participation than in-person meetings where participation is often fluctuating based on the location.
-
-However, it has also been recognised that the online meeting does not provide an equivalent opportunity to socialize.
-The observed slight decrease in submission of new (-00) drafts during 2020 and 2021, while the overall number of draft submissions and
-productivity seem to stay stable, might also be an indication of the loss of these interactions. The increase in interim
-meetings potentially compensates for these missing interactions for continuous work (or may even increase
-productivity there), but seems to be less adequate to spark new ideas.
-
-None of the data observed, however, can be interpreted as showing a significant trend. Still, these factors
-should be considered for the organization of future online only meetings in replacement or addition to in-person meetings.
+This document RECOMMENDS further experiments with reducing conflicts by
+leveraging the increased flexibility of the online format.
+
+An in-person meeting must fit all sessions into an acceptable length for
+international travel (usually roughly a week), but online meetings do not have
+that constraint.
+
+Therefore, it would be possible to keep most regular working group sessions
+within the usual five main meeting days but have some of the more conflicted
+sessions in other dedicated time slots. As the Hackathon for online only
+meetings is usually held in the week before the online plenary meeting
+{{RFC9311}}, that week is already a highly active week for many IETF
+participants and might provide an opportunity to schedule a few selected
+sessions.
+
+This might work especially well for sessions that are of high interest for a
+large part of community, such as BoFs and dispatch meetings, and therefore hard
+to schedule during the main IETF week.
+
+At IETF 112, the IESG ran an experiment where the adminstrative plenary was
+scheduled on the Wednesday before the official session week. The experiment
+report {{112-EXPERIMENT}} found that it led to a reduction in scheduling
+conflicts but also a slight drop in plenary attendance, partly due to
+insufficient awareness.
+
+## Inclusivity and Socializing
+
+Participation at the online only meetings in 2021 was high and had a stable
+per-country distribution, even though time zones were rotated. This indicates
+that online meetings support a more consistent geographic distribution of
+participants than in-person meetings, where participation often fluctuates based
+on the location.
+
+However, online meetings do not provide an equivalent opportunity to socialize.
+Despite significant investment in tools to foster hallway conversations,
+participation was lower than at an in-person meeting, whether due to ignorance
```

Comment:
participation was lower compared to what? Fully online (except 107) was usually around 1200-1400. Before the pandemic we were at in-person+remote at 1400-2000 but remote attendance was fully free, so that not completely comparable. Since we are back we are at around 1400-1500. So in average we where lower for fully online but it there have been many changes over the last 3 years, so it's actually hard to compare and draw a conclusion.

Reply:
No, I was referring to presence in Gather, not the meeting as a whole. Rephrased to be clearer.

Reply:
Ah, okay, new text is clear now but I'm not sure if "many did not use those tools" is that objectively true either... maybe something like "hallway-like interactions were significantly lower compared to in-person meetings"...?